salsa_dancing_molecules/lennardjonesparse.py

```diff
@@ -12,12 +12,15 @@ def parse_lj_params(atom_letters):
     """
     with open('LennardJones612_UniversalShifted.params') as param_file:
         lines = param_file.readlines()[7:]
+        element_number = 1
```

Comment:
This assumes the elements are sorted in the parameter list. Maybe we should add a comment to the parameter file stating that the order is important?